Sources/Shared/Protocols/PathAccessible.swift

```diff
@@ -40,7 +34,15 @@ public extension PathAccessible {
     return result as? T
   }
 
-  private func resolve<T>(path: String) -> T? {
+  private func resolveSubscript<T>(key: String) -> T? {
+    if let index = Int(key) {
+      return [index] as? T
```

Comment:
`T` is the value you expect, it subscripts the value inside of the array, hence `[index]`.